src/CultureFeed/CultureFeedApiKeyAuthenticator.php

```diff
@@ -27,27 +27,24 @@ class CultureFeedApiKeyAuthenticator implements ApiKeyAuthenticatorInterface
     private $includePermissions;
 
     /**
-     * @param \ICultureFeed $cultureFeed
-     * @param ConsumerWriteRepositoryInterface $consumerWriteRepository
      *   CultureFeed returns a consumer detail while authenticating.
      *   This consumer will be written to the injected write repository as a
      *   form of caching.
```

Comment:
Nit: the description that's now left was for the removed `@param`. If we leave it in, we should probably change the indentation?

Reply:
This is part of a merge from what's already in `main`, so probably better to fix it there immediately.

Reply:
Yeah nevermind it's super minor and not worth it then 😄